src/styles/text.tsx

```diff
@@ -11,3 +15,23 @@ export const H2 = styled.h2`
   font-size: 1.5rem;
   margin: 0;
 `;
+
+export const H4 = styled.h4`
+  color: ${COLORS.existingInputColor};
+  ${openSans.style}
+  font-size: 1rem;
+  font-style: normal;
+  font-weight: 600;
+  line-height: normal;
+`;
+
+export const H4Centered = styled(H4)`
+  text-align: center;
+`;
+
+export const P = styled.p`
+  ${openSans.style}
+  font-size: 0.875rem;
+  font-weight: 400;
+  color: ${COLORS.dark};
+`;
```

Comment:
Same thing here as my other PR, needed global styles for text so I just went ahead and copied over what Catherine ++ added in her PR.